### PR TITLE
Feature/accessibility alt tag for capture previews cx 3476

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
As part of the accessibility work, we want to make the capture previews accessible to screen readers

# Solution
Added alt tags for capture previews. PDF previews are out of scope

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [x] Have tests passed locally?
- [x] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-645-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-645-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-645-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://645-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by joninvski** [comment]
@stefaniacardenas what do you think about squashing `Remove changes for PDF` and `Added alt tag for images. Handle PDF still in progress`.

SInce the squashed commit would have a single scope, it would help future read of the commit history.

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Public: Accessibility - Make H1 readable by screen readers
 - Public: Accessibility - Make buttons/links readable by screen readers, allow tabbing to them
 - Public: Accessibility - Announce page transition when screen changes
+- Public: Accessibility - Make capture previews readable by screen readers
```
Comment (rfreitas) [inline CHANGELOG.md]: Accessibility is actually UI

Comment (josokinas) [reply CHANGELOG.md]: @rfreitas Should we have a separate PR, or fix that when release happens?

Comment (josokinas) [reply CHANGELOG.md]: @rfreitas Fixed (previous entries) with a PR. @stefaniacardenas please if you could sync the branch and change your entry.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-645-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-645-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://645-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-645-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-645-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://645-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-645-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-645-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://645-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**